apps/asap-server/src/utils/squidex.ts

```diff
@@ -12,3 +12,6 @@ export const createURL = (assets: string[]): string[] =>
 
 export const parseDate = (date: string): Date =>
   DateTime.fromISO(date).toJSDate();
+
+export const sanitiseForSquidex = (text: string): string =>
+  escape(text.replace(/'/g, "''"));
```

Comment:
https://www.w3schools.com/jsref/jsref_escape.asp isn't this deprecated?

Reply:
![image](https://user-images.githubusercontent.com/11645956/116529331-c06f8800-a8d4-11eb-9ce6-482ed6f96dd9.png)

Reply:
the reason i used it is because there doesn't seem to be any package that does what we want. Well there is xss-filters but i think its a bit of an overkill.